src/test/kotlin/com/autonomousapps/internal/advice/ConsoleReportTest.kt

```diff
@@ -0,0 +1,305 @@
+    assertEquals(expectedImpl, actualImpl, "Expected $expectedImpl\nActual   $actualImpl\n")
+  }
+
+  @Test fun `from should not return the change advices from a computed analysis with a filter`() {
+    // Given
+    val filterSpecBuilder = FilterSpecBuilder()
+    filterSpecBuilder.incorrectConfigurationsBehavior = Ignore
+    val computedAdvice = ComputedAdvice(
+      unusedDependencies = emptySet(),
+      undeclaredApiDependencies = emptySet(),
+      undeclaredImplDependencies = emptySet(),
+      changeToApi = setOf(orgDotSomethingV1),
+      changeToImpl = emptySet(),
+      filterSpecBuilder= filterSpecBuilder,
+      compileOnlyCandidates = emptySet(),
+      unusedProcs = emptySet()
+    )
+
+    // When
+    var consoleReport = ConsoleReport.from(computedAdvice = computedAdvice)
+
+    // Then
+    val adviceApi = consoleReport.changeToApiAdvice
+    assertTrue(adviceApi.isEmpty(), "Expected $adviceApi to be empty")
+    val adviceImpl = consoleReport.changeToImplAdvice
+    assertTrue(adviceImpl.isEmpty(), "Expected $adviceImpl to be empty")
+  }
+
+  @Test fun `from should return the compileOnly advices from a computed analysis`() {
+    // Given
+    val filterSpecBuilder = FilterSpecBuilder()
+    val computedAdvice = ComputedAdvice(
+      unusedDependencies = emptySet(),
+      undeclaredApiDependencies = emptySet(),
+      undeclaredImplDependencies = emptySet(),
+      changeToApi = emptySet(),
+      changeToImpl = emptySet(),
+      filterSpecBuilder= filterSpecBuilder,
+      compileOnlyCandidates = setOf(componentV1),
+      unusedProcs = emptySet()
+    )
+
+    // When
+    var consoleReport = ConsoleReport.from(computedAdvice = computedAdvice)
+
+    // Then
+    val expected = setOf(
+      Advice.compileOnly(orgDotSomethingV1, "compileOnly")
+    )
+    val actualApi = consoleReport.compileOnlyDependencies
+    assertEquals(expected, actualApi, "Expected $expected\nActual   $actualApi\n")
+  }
+
+  @Test fun `from should not return the compileOnly advices from a computed analysis with a filter`() {
+    // Given
+    val filterSpecBuilder = FilterSpecBuilder()
+    filterSpecBuilder.compileOnlyBehavior = Ignore
+    val computedAdvice = ComputedAdvice(
+      unusedDependencies = emptySet(),
+      undeclaredApiDependencies = emptySet(),
+      undeclaredImplDependencies = emptySet(),
+      changeToApi = emptySet(),
+      changeToImpl = emptySet(),
+      filterSpecBuilder= filterSpecBuilder,
+      compileOnlyCandidates = setOf(componentV1),
+      unusedProcs = emptySet()
+    )
+
+    // When
+    var consoleReport = ConsoleReport.from(computedAdvice = computedAdvice)
+
+    // Then
+    val adviceApi = consoleReport.changeToApiAdvice
+    assertTrue(adviceApi.isEmpty(), "Expected $adviceApi to be empty")
+    val adviceImpl = consoleReport.changeToImplAdvice
+    assertTrue(adviceImpl.isEmpty(), "Expected $adviceImpl to be empty")
+  }
+
+  private fun createReport(addToApiAdvice: Set<Advice> = emptySet(),
```

Comment:
please fix stye. Each param on its own line, including the first. That will also fix the indentation. This much whitespace hurts my brain :)